internal/config/network.go

```diff
@@ -51,6 +57,9 @@ var TestNet3 = Network{
 	// DCP0010Height on testnet3 is block
 	// 000000000000c7fd75f2234bbff6bb81de3a9ebbd2fdd383ae3dbc6205ffe4ff.
 	DCP0010Height: 877728,
+	// DCP0012Height on testnet3 is block
+	// c7da7b548a2a9463dc97adb48433c4ffff18c3873f7e2ae99338a990dae039f0.
```

Comment:
Confirmed

```sh
$ dcrctl --testnet getblockchaininfo | jq -c '.deployments.changesubsidysplitr2 | {status,since}'
{"status":"active","since":1170048}

$ dcrctl --testnet getblockchaininfo | jq '.deployments.changesubsidysplitr2.since' | dcrctl --testnet getblockhash -
c7da7b548a2a9463dc97adb48433c4ffff18c3873f7e2ae99338a990dae039f0
```